src/index.js

```diff
@@ -2,18 +2,18 @@ import * as plugin from "./plugin";
 import CldContext from "./components/CldContext/CldContext";
 import CldImage from "./components/CldImage/CldImage";
 import CldVideo from "./components/CldVideo/CldVideo";
+import CldPoster from './components/CldVideo/CldPoster';
+import CldTransformation from './components/CldTransformation/CldTransformation';
 
-const components = {
-  CldContext,
-  CldImage,
-  CldVideo
+const allComponents = { CldContext, CldImage, CldVideo, CldTransformation, CldPoster };
```

Comment:
`allComponents` is not used outside of `install` function call, so it'd be more readable to place it there.

I wouldn't, however, even point that out, but there is also potential issue with pre-tree-shaking static code analysis. I'd suspect that named imports could fail to import just a single export (which should be easy as this file could really only just reexport other file exports) as this variable uses all of them and is not what it is meant to be without all components. Might be that minifiers could also reject that variable if `install` function declared here was not used and rejected also, but I'm not sure of that.

Reason for wanting to be simple in the first place was that we are not in control of any tooling that is and will be, so cannot guarantee that those tools will be capable of such smart conclusions like when to strip cloudinary-vue bundle of unnecessary components even if in this entry file, in this line they are all used.

Keeping this file containing just reexports had this advantage that it was simple to conclude which files should be attached to an app bundle for a static code analysis tool. That is why plugin file was importing those files. As I understand you want to make users to import each component and use that to specify installation, so plugin may not import cld* components in this setup.

Reply:
The  best way to tree shaking is to import dynamically the component upon request (all Components), but I haven't figured out how to do it yet without configure webpack further so for now it should be ok :).